test/e2e/etcd_backup_test.go

```diff
@@ -146,12 +148,12 @@ var _ = Describe("Etcd Backup", func() {
 							return fmt.Errorf("sts %s is still in ready state", etcdName)
 						}
 						return nil
-					}, timeout*3, pollingInterval).Should(BeNil())
+					}, singleNodeEtcdTimeout, pollingInterval).Should(BeNil())
 					logger.Info("sts is unready", "statefulSetName", etcdName)
 
 					logger.Info("waiting for sts to become ready again", "statefulSetName", etcdName)
 					Eventually(func() error {
-						ctx, cancelFunc := context.WithTimeout(context.TODO(), timeout)
+						ctx, cancelFunc := context.WithTimeout(context.TODO(), singleNodeEtcdTimeout)
```

Comment:
**Nit pick:**
I have seen that there is a general confusion on when to use `context.TODO` and when to use `context.Background`. Both of these return an empty context and the difference is more semantic.
If you look at the documentation for these 2 methods then it will become clear https://github.com/golang/go/blob/master/src/context/context.go#L204-L218
Given the description i would suppose that context.Background would be semantically more appropriate.

Reply:
Fixed